actionpack/test/dispatch/executor_test.rb

```diff
@@ -127,12 +103,15 @@ def test_body_abandoned
     executor.to_run { total += 1; ran += 1 }
     executor.to_complete { total += 1; completed += 1 }
 
-    stack = middleware(proc { [200, {}, "response"] })
+    app = proc { [200, {}, []] }
+    env = Rack::MockRequest.env_for("", {})
+
+    stack = Rack::Lint.new(middleware(Rack::Lint.new(app)))
```

Comment:
What do you think about moving the `Rack::Lint` wrapping into the `middleware` method? Then we'd only be doing the wrapping in one place instead of two